src/sInv.sol

```diff
@@ -306,14 +323,15 @@ contract sINV is ERC4626 {
      * @param amount The amount of tokens to sweep.
      * @param to The recipient address of the swept tokens.
      */
-    function sweep(address token, uint amount, address to) public onlyGov {
-        require(address(DBR) != token, "Not authorized");
-        require(address(asset) != token, "Not authorized");
+    function sweep(address token, uint256 amount, address to) public onlyGov {
+        if(address(DBR) == token ||
+            address(asset) == token)
+            revert UnauthorizedTokenWithdrawal();
         IERC20(token).transfer(to, amount);
     }
 
-    event Buy(address indexed caller, address indexed to, uint exactInvIn, uint exactDbrOut);
-    event SetTargetK(uint newTargetK);
-    event SetPeriod(uint newPeriod);
-    event SetMinBuffer(uint newMinBuffer);
+    event Buy(address indexed caller, address indexed to, uint256 exactInvIn, uint256 exactDbrOut);
+    event SetTargetK(uint256 newTargetK);
+    event SetPeriod(uint256 newPeriod);
```

Comment:
Unused event

```suggestion
```

Reply:
Fixed in [7fc3885](https://github.com/InverseFinance/sINV/pull/2/commits/7fc388516d12a13f200690249e8d02264ea31d5f)